tools/cldr-apps/js/src/esm/cldrGui.js

```diff
@@ -416,6 +416,7 @@ function handleResize() {
 }
 
 function updateWithStatus() {
+  cldrProgress.fetchLocaleData(true);  // skip if already loaded
   mainHeaderWrapper?.updateData();
 }
```

Comment:
i think updateWithStatus is only called once per session

another good place for it might be updateCompletionOneVote

but how expensive (time-consuming) is the request? fetching the whole data set too often might be problematic

Reply:
updateWithStatus() is called periodically, after what=status fetch.

> another good place for it might be updateCompletionOneVote

Added that in 57c58d4

Reply:
> but how expensive (time-consuming) is the request

It should be cached. But in any event, it also won't change very often if the user isn't themselves voting.